pkg/config/options.go

```diff
@@ -5,7 +5,7 @@ import (
 )
 
 type ApiOptions struct {
-	Port int `short:"p" long:"port" description:"Port to listen on" default:"5050"`
+	Port int `short:"p" long:"port" description:"Port to listen on" default:"80"`
```

Comment:
I know we talked about 80 earlier, but for defaults that get used for local dev we should really use a unique port. If every XMTP service defaulted to 80, we'd constantly have port conflicts doing anything locally.

Totally OK with setting it to 80 in the production service, which we can do with the `--port` command line argument at runtime.